tests/pytest/test_multi.py

```diff
@@ -227,6 +227,8 @@ def testSetAndGetCommands(env):
     res = r.execute_command('JSON.GET', 'doc1')
     r.assertEqual(res, '"inizio"')
 
+    # Test unicode characters
+    r.assertOk(r.execute_command('JSON.SET', 'doc1', '$', '"\u00a0"'))
```

Comment:
Can also confirm we can get it back with `JSON.GET`

Reply:
I'll actually remove it from here, this is not needed there.

Reply:
I apologise for this one. I just quickly looked for the first test file I could open in my editor and added a line there to confirm it happens/doesn't happen. This test should be a separate thing, so I removed these two lines from here.